packages/grid/_modules_/grid/components/toolbar/ExportSelector.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+import MenuList from '@material-ui/core/MenuList';
+import Button from '@material-ui/core/Button';
+import MenuItem from '@material-ui/core/MenuItem';
+import { ApiContext } from '../api-context';
+import { useGridSelector } from '../../hooks/features/core/useGridSelector';
+import { optionsSelector } from '../../hooks/utils/optionsSelector';
+import { GridMenu } from '../menu/GridMenu';
+import { ExportOption } from '../../models';
+
+export function ExportSelector() {
+  const apiRef = React.useContext(ApiContext);
+  const options = useGridSelector(apiRef, optionsSelector);
+  const [anchorEl, setAnchorEl] = React.useState(null);
+
+  const ExportIcon = apiRef!.current.components!.ExportIcon!;
+
+  const ExportOptions: Array<ExportOption> = [
+    {
+      label: apiRef!.current.getLocaleText('toolbarExportCSV'),
+      format: 'csv',
+    },
+  ];
+
+  const handleExportSelectorOpen = (event) => setAnchorEl(event.currentTarget);
+  const handleExportSelectorClose = () => setAnchorEl(null);
+  const handleExport = (format) => {
+    if (format === 'csv') {
+      apiRef!.current.exportDataAsCsv();
+    }
+
+    setAnchorEl(null);
+  };
+
+  const handleListKeyDown = (event: React.KeyboardEvent) => {
+    if (event.key === 'Tab' || event.key === 'Escape') {
+      event.preventDefault();
+      handleExportSelectorClose();
+    }
+  };
```

Comment:
A side note, I think that we will need to move MenuList inside GridMenu. Having to duplicate this logic is signal that the abstraction is wrong. A `onClose` even should be enough. It will also help drop GridMenu once we fix the underlying component in the core.

Reply:
Agree, I also noticed the duplicate logic. I'll tackle the focus problem first and check this one.

Reply:
After checking it seems that for now, I'll have to keep it as it is because this is also used for the component slot in the column menu. This means that If I move the `MenuList` part in the `GridMenu` then whoever wants to overwrite the column menu will still have to use `MenuList` as a parent.

We can have a separate ticket to track and tackle this problem alone.

Reply:
Agree with the separate ticket part. It's a larger problem to CSV.